watcher_test.go

```diff
@@ -34,6 +39,15 @@ func TestWatcher(t *testing.T) {
 	w.Close()
 	time.Sleep(time.Millisecond * 500)
 }
+func TestWatcherWithIgnoreSelf(t *testing.T) {
+	_, w := initWatcherWithOptions(t, WatcherOptions{IgnoreSelf: true})
+	_ = w.SetUpdateCallback(func(s string) {
+		fmt.Println(s)
```

Comment:
This not a test, plz take other tests as examples.

Reply:
Fixed.